static/app/views/alerts/rules/issue/index.tsx

```diff
@@ -905,7 +905,9 @@ class IssueRuleEditor extends AsyncView<Props, State> {
           value={this.getTeamId()}
           project={project}
           onChange={this.handleOwnerChange}
-          teamFilter={(team: Team) => team.isMember || team.id === ownerId}
+          teamFilter={(team: Team) =>
+            team.isMember || team.id === ownerId || team.access.includes('team:admin')
```

Comment:
can someone not be member but be admin?

Reply:
Yes. Org-owners will get `team:admin` scope without being member.

Reply:
I must check for `isMember` because non-members will get `team:read` too.